app/konnect/index.md

```diff
@@ -20,49 +30,57 @@ each runtime group.
 
 ### Service Hub
 
-The [Service Hub](/konnect/servicehub) enables the cataloging of all services
-into a single system of record. This catalog represents the single source of
-truth for your organization’s service inventory and their dependencies. By
-leveraging Service Hub, application developers can search, discover, and consume
-existing services to accelerate their time-to-market, while enabling a more
-consistent end-user experience across the organization’s applications.
+[Service Hub](/konnect/servicehub) makes internal APIs discoverable, 
+consumable, and resuable for internal development teams. Service Hub 
+enables the cataloging of all your services into a single system of record.
+This catalog represents the single source of truth for your organization’s 
+service inventory. By leveraging Service Hub, your application developers 
```

Comment:
```suggestion
[Service Hub](/konnect/servicehub) makes internal APIs discoverable, 
consumable, and reusable for internal development teams. Catalog
all your services through the Service Hub to create a single source of 
truth for your organization’s service inventory. By leveraging Service Hub, 
your application developers 
```
Combined the two "catalog" sentences to avoid repeating "single system of record" and "single source of truth", since they mean the same thing.